tests/backbeat/Using.md

```diff
@@ -100,10 +100,17 @@
    and `.secrets.env`):
```

Comment:
I'd also add the steps needed to create the Azure destination location. I think these steps would be the same as those for creating Azure resources for CRR tests—probably just copying over those steps will do. 😸